README.md

```diff
@@ -3,146 +3,120 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg?style=flat-square)](/CODE_OF_CONDUCT.md)
 
-Json.NET is a popular high-performance JSON framework for .NET
+Json.<i></i>NET is a popular high-performance JSON framework for .NET
 
-This repo is a **fork** of [Newtonsoft.Json][newtonsoft.json.git] containing custom builds
+Available for installation with
+
+- **Unity Package Manager (UPM)** `new!`
+- ~~UniGet~~ <sub>_(Coming soon<sup>tm</sup>)_</sub>
+- ~~Unity Asset packages~~ <sub>_(Coming soon<sup>tm</sup>)_</sub>
+- ~~Unity Asset Store~~ <sub>_(Coming soon<sup>tm</sup>)_</sub>
+
+This repo is a **fork** of [JamesNK/Newtonsoft.Json][newtonsoft.json.git] containing custom builds
 for regular standalone, but more importantly AOT targets such as all **IL2CPP builds**
 **(WebGL, iOS, Android, Windows, Mac OS X)** and portable .NET **(UWP, WP8)**.
 
-Available for installation with
+## [Installation guide](https://github.com/jilleJr/Newtonsoft.Json-for-Unity/wiki/Installation-via-UPM)
 
-- **Unity Package Manager (UPM)** `new!`
-- ~~Unity Asset packages~~ _(coming soon)_
-- ~~Unity Asset Store~~ _(coming soon)_
+Click the header. It goes to the Wiki where the guide is now located. For reference, this is the url:
+
+<https://github.com/jilleJr/Newtonsoft.Json-for-Unity/wiki/Installation-via-UPM>
+
+## Newtonsoft.Json-for-Unity specific links
+
+- [Wiki about this project](https://github.com/jilleJr/Newtonsoft.Json-for-Unity/wiki)
+- [Release Notes on GitHub from this repository](https://github.com/jilleJr/Newtonsoft.Json-for-Unity/releases)
+- [Cloudsmith package](https://cloudsmith.io/~jillejr/repos/newtonsoft-json-for-unity/packages/detail/npm/jillejr.newtonsoft.json-for-unity/latest/)
+
+## Official Json.<i></i>NET links
+
+- [Source repository (github.com/JamesNK/Newtonsoft.Json)](https://github.com/JamesNK/Newtonsoft.Json)
+- [Homepage (www.newtonsoft.com/json)](https://www.newtonsoft.com/json)
+- [Documentation (www.newtonsoft.com/json/help)](https://www.newtonsoft.com/json/help)
+- [Release Notes on GitHub from source repository](https://github.com/JamesNK/Newtonsoft.Json/releases)
+- [Stack Overflow posts tagged with `json.net`](https://stackoverflow.com/questions/tagged/json.net)
+
+## Contributing
+
+> ⚠ Please note that this project is released with a Contributor Code of Conduct.
+> By participating in this project you agree to abide by its terms.
+> Read more: [CODE_OF_CONDUCT.md](/CODE_OF_CONDUCT.md)
+
+To contribute first you must read the [CONTRIBUTING.md](/CONTRIBUTING.md)
+guidelines. It contains info about
+
+- How to edit the Src/Newtonsoft.Json projects to remain the ability to merge
+  from JamesNKs repository without difficulties.
+- Coding style.
+- Naming style.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743215)